app/controllers/comments_controller.rb

```diff
@@ -1,2 +1,16 @@
 class CommentsController < ApplicationController
+  def create
+    @publisher = User.find(params[:user_id])
```

Comment:
Please make sure that the comments controller has the `new`  action for creating a new comment.
 i.e
``` 
def new
...
  end
```